src/containers/User/ClubListItem/index.js

```diff
@@ -1,35 +1,65 @@
 import React from 'react'
 import { inject, observer } from 'mobx-react'
+import { observe } from 'mobx'
 import { Link } from 'react-router-dom'
 import { Flex, } from 'antd-mobile'
 import './clubListItem.css'
 
-@inject('clublistStore') @observer
+@inject('clubListStore') @observer
 class ClubListItem extends React.Component {
 
+  constructor(props) {
+    super(props)
+    this.state = this.props.clubListStore
+  }
+
+  componentDidMount() {
+    this.state.getUserClubs(10)
+  }
+
   render() {
     const {
       handleClubDetail,
       clubDataList,
       clubListArrow,
       maxAvatars,
-    } = this.props.clublistStore
+      userClubList,
+      clubNameList,
+      clubIdList,
+      clubUsers,
+      clubUserAvatars,
+    } = this.state
+
+    let clubData = []
+    if (clubIdList.length === clubNameList.length) {
+      for (let i = 0; i < clubIdList.length; i++) {
+        clubData.push({
+          clubName: clubNameList[i],
+          avatar: clubUserAvatars.slice().map(proxy => proxy.slice()),
```

Comment:
what does `slice()` do?

Reply:
`slice()` is to convert Proxy to values, every observable in store is saved as proxy.